missing description and a language tweak (#1898)
filecoin-project/filecoin-docs: content/en/storage-provider/basics/overview/index.md

---
title: "Becoming a Storage Provider"

> description: "Understand the key concepts you need in order to design a suitable architecture, make the right hardware investments, and run a profitable Storage Provider business"

Reviewer: ```suggestion
description: "This page will help you understand how to design a suitable storage provider architecture, make the right hardware investments, and run a profitable business."
```
